packages/lib/storybook/stories/internals/Alert.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Meta, StoryObj } from '@storybook/preact';
+import Alert from '../../../src/components/internal/Alert';
+import Language from '../../../src/language';
+import { CoreProvider } from '../../../src/core/Context/CoreProvider';
+import { Resources } from '../../../src/core/Context/Resources';
+
+const meta: Meta = {
+    title: 'Internals/Alert',
+    component: Alert,
+    argTypes: {
+        type: {
+            options: ['error', 'warning', 'info', 'success'],
+            control: { type: 'radio' }
+        },
+        icon: {
+            options: ['cross', 'info', 'info', 'success'],
+            control: { type: 'radio' }
+        }
+    }
+};
+
+export const Default: StoryObj = {
+    render: args => {
+        return (
+            <CoreProvider
+                loadingContext={'test'}
+                i18n={
+                    new Language({
+                        locale: 'en-US',
+                        translations: {}
+                    })
+                }
+                resources={new Resources('https://checkoutshopper-test.cdn.adyen.com/checkoutshopper/')}
+            >
+                <Alert {...args}>dummy alert message</Alert>
```

Comment:
Interesting that a simple UI component has a direct dependency on having the CoreProvider + Resources + Language modules in order for it to work. Perhaps this needs to be more decoupled at some point.